.github/workflows/lint.yml

```diff
@@ -32,4 +37,4 @@ jobs:
         uses: wearerequired/lint-action@v1
         with:
           eslint: true
-          prettier: true
+      # prettier: true
```

Comment:
Instead of commenting out, if we don't need it, just delete it.

Reply:
@git-bd1599 please fix

Reply:
Have deleted prettier for now.